src/Pages/Categories.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Categories = () => <button type="button">Check Status</button>;
+
+export default Categories;
```

Comment:
- [ ] It is required that your categories component contains a check status button and when clicked, it should dispatch the check-status action creator which will display the payload to the UI. Kindly look into that and fix. Please, take a look :point_down: 


![proj-rqt](https://user-images.githubusercontent.com/26597432/216710795-1e1bd55f-30c9-4d24-8135-524264abdec7.png)